docs/ota-client-guide/modules/ROOT/nav.adoc

```diff
@@ -33,7 +33,7 @@ ifndef::env-github[:pageroot:]
 * xref:{pageroot}generating-provisioning-credentials.adoc[Get a provisioning Key]
 * xref:{pageroot}build-images.adoc[Use our sample recipes to build disk images]
 ** xref:{pageroot}build-raspberry.adoc[Build for a Raspberry Pi]
-** xref:{pageroot}build-quemu.adoc[Build for QEMU]
+** xref:{pageroot}build-qemu.adoc[Build for QEMU]
```

Comment:
@patrickvacek FYI: Changing a file name will break an include in another repo (the "Get Started" repo) - let me fix that before you merge.

Reply:
Ah, sorry. Let me know when you're ready.

Reply:
Alright done, will click "Approve" now.

Reply:
Thanks!